basics/script.js

```diff
@@ -0,0 +1,8 @@
+function fullName() {
```

Comment:
Where are we using this?

Reply:
sir , this function is used to concat the first name and the last name and fill the full name field